Library/Formula/mplayer.rb

```diff
@@ -51,18 +40,17 @@ def install
     ENV.O1 if ENV.compiler == :llvm
 
     # we disable cdparanoia because homebrew's version is hacked to work on OS X
-    # and mplayer doesn't expect the hacks we apply. So it chokes.
+    # and mplayer doesn't expect the hacks we apply. So it chokes. Only relevant
```

Comment:
I thought it'd be wise to then maybe make it more clear when this is a relevant option (alas when you have installed via homebrew). This way, this doesn't come up next time someone cleans up the formula